configs/agent_config.go

```diff
@@ -53,6 +55,72 @@ type AgentHooks struct {
 	DoOnWorkflowEnd   string `yaml:"do_on_workflow_end"`
 }
 
+func RegisterAgentOverrides() error {
+	if !hasAgentConfigFile() {
+		return nil
+	}
+
+	file := filepath.Join(GetBitriseHomeDirPath(), agentConfigFileName)
+
+	log.Print("")
+	log.Info("Running in agent mode")
+	log.Printf("Config file: %s", file)
+
+	config, err := readAgentConfig(file)
+	if err != nil {
+		return fmt.Errorf("agent config file: %w", err)
+	}
+	log.Printf("env: %s", os.Getenv(BitriseDataHomeDirEnvKey))
+
+	// BITRISE_DATA_HOME_DIR
+	err = pathutil.EnsureDirExist(config.BitriseDirs.BitriseDataHomeDir)
+	if err != nil {
+		return fmt.Errorf("can't create BITRISE_DATA_HOME_DIR: %w", err)
+	}
+	err = os.Setenv(BitriseDataHomeDirEnvKey, config.BitriseDirs.BitriseDataHomeDir)
+	if err != nil {
+		return fmt.Errorf("set BITRISE_DATA_HOME_DIR: %w", err)
+	}
+
```

Comment:
super nit: there are 2 newlines here while everywhere else in the same file there is only 1.

Reply:
Fixed